jsonrpclib/SimpleJSONRPCServer.py

```diff
@@ -1,19 +1,26 @@
 import jsonrpclib
 from jsonrpclib import Fault
 from jsonrpclib.jsonrpc import USE_UNIX_SOCKETS
-import SimpleXMLRPCServer
-import SocketServer
+try:
```

Comment:
You may think about reversing these, since Python 3.x will be the default, and these options will be less likely (over time) to be what the system has installed.